shopsafe-backend/src/main/java/com/google/sps/servlets/StoresServlet.java

```diff
@@ -38,22 +49,53 @@
 @WebServlet("/stores")
 public class StoresServlet extends HttpServlet {
 
+    public static final String PLACE_URL = "https://maps.googleapis.com/maps/api/place/nearbysearch/json?location=";
+    public static final String PLACE_TYPE = "&radius=12000&type=grocery_or_supermarket";
+    private String PLACE_KEY;
+
     /**
      * For a get request, return all nearby stores world.
```

Comment:
I'm a little confused by this wording: what does nearby stores world mean?

Reply:
Oh my, I really don't know why the word world is there.